src/common/resources/contacts.js

```diff
@@ -15,3 +15,16 @@ export const baseContactsResource = {
   records: 'contacts',
   path: CONTACTS_API,
 };
+
+export const baseInterfacesResource = {
```

Comment:
Firstly, it looks something wrong with naming. interface and interfaces are already different names, what `base` means here?
Secondly, I don't think it should be in contacts.js